Source/Scene/Cesium3DTileset.js

```diff
@@ -945,73 +1045,50 @@ define([
             });
 
             while (stack.length > 0) {
-                var t = stack.pop();
-                var tile3D = t.cesium3DTile;
-                var children = t.header.children;
-                var hasEmptyChild = false;
+                var tile = stack.pop();
+                var tile3D = tile.cesium3DTile;
+                var children = tile.header.children;
                 if (defined(children)) {
                     var length = children.length;
                     for (var k = 0; k < length; ++k) {
                         var childHeader = children[k];
                         var childTile = new Cesium3DTile(that, baseUrl, childHeader, tile3D);
                         tile3D.children.push(childTile);
+                        childTile._depth = tile3D._depth + 1;
                         ++stats.numberTotal;
                         stack.push({
                             header : childHeader,
                             cesium3DTile : childTile
                         });
-                        if (!childTile.hasContent) {
-                            hasEmptyChild = true;
-                        }
                     }
                 }
                 Cesium3DTileOptimizations.checkChildrenWithinParent(tile3D, true);
-                if (tile3D.hasContent && hasEmptyChild && (tile3D.refine === Cesium3DTileRefine.REPLACE)) {
-                    // Tiles that use replacement refinement and have empty child tiles need to keep track of
-                    // descendants with content in order to refine correctly.
-                    refiningTiles.push(tile3D);
+
+                // Create a load heap, one for each unique server. We can only make limited requests to a given
```

Comment:
@lilleyse should we make a small tweak to the request schedule now to let it know that these are priority even though we're not making a ton of requests?